test/sql/udf_function/udf_functions_to_test.hpp

```diff
@@ -0,0 +1,54 @@
+/*HEADER file with all UDF Functions to test*/
+#include "duckdb/common/types/string_type.hpp"
+
+#pragma once
+using namespace duckdb;
```

Comment:
No `using namespace` in a header please :) And `#pragma once` should appear on top.